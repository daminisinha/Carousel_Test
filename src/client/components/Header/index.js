import React, { Component } from 'react';
import './index.scss';
class Header extends Component {
    render() {
        return (
            <div className="header">
            <h1>Carousel</h1>
            </div>
        );
    }
}

export default Header;