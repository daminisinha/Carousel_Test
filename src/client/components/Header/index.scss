@import '../../utilities/variables';
h1{
    color:$app-color;
    padding-left:20px;
    margin: 5px;
}
.header{
    height:40px;
}