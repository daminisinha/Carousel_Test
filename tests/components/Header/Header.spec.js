import React from 'react';
import Header from '../../../src/client/components/Header'
import {shallow} from 'enzyme';

test('Header renders',()=>{
    const component = shallow(<Header/>); 
    expect(component.find('h1')).toHaveLength(1);
    expect(component).toMatchSnapshot();
})
