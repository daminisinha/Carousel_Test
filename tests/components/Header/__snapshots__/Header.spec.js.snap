// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<div
  className="carousel_header"
>
  <h1>
    Carousel Test
  </h1>
</div>
`;
